packages/connectivity/src/scp-cf/resilience-options.ts

```diff
@@ -1,15 +1,18 @@
 /**
  * Options to configure resilience when fetching destinations.
+ * @deprecated  Since v2.13.0. The resilience configuration will be removed in version 3.0 of the SDK and is switched on per default.
  */
 export interface ResilienceOptions {
   /**
    * A boolean value that indicates whether to execute request to SCP-CF services using circuit breaker.
    * ResilienceOptions.
+   * @deprecated   Since v2.13.0. Option will be removed in version 3.0 of the SDK. Circuit breaker is enabled per default and it can not be disabled anymore.
```

Comment:
```suggestion
   * @deprecated Since v2.13.0. Option will be removed in version 3.0 of the SDK. Circuit breaker is enabled per default and it can not be disabled anymore.
```